examples/sand_and_gravel/12345678901.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
The file names should stay the same as they represent the schema versions. If we wanted multiple examples for each schema version, then they should be in a folder with the version name.

so either `v1.json` or `v1/<file_name>.json` (if we wanted multiple examples)